lib/internal/util/types.js

```diff
@@ -2,10 +2,18 @@
 
 const {
   ArrayBufferIsView,
+  ArrayBufferPrototype,
   ObjectDefineProperties,
+  ReflectGet,
   TypedArrayPrototypeGetSymbolToStringTag,
 } = primordials;
 
+const { isArrayBufferDetached: _isArrayBufferDetached, ...internalTypes } = internalBinding('types');
+
+function ArrayBufferGetByteLength(view) {
+  return ReflectGet(ArrayBufferPrototype, 'byteLength', view);
+}
```

Comment:
Why not use `primordials.ArrayBufferPrototypeGetByteLength` directly?

Reply:
You're right. I'm updating the pull request, and fixing webstreams in a different pull request.